source/javascripts/meetups.js

```diff
@@ -44,37 +64,35 @@
   locations.forEach( generateMarkerData );
 
   handler.buildMap(mapOptions, function() {
```

Comment:
It looks like he did something along the same lines, which I had missed. How should I proceed?